actix-web-actors/src/ws.rs

```diff
@@ -279,6 +434,33 @@ where
         (addr, WebsocketContextFut::new(ctx, actor, mb, Codec::new()))
     }
 
+    #[inline]
+    /// Create a new Websocket context from a request, an actor, and a codec.
+    ///
+    /// Returns a pair, where the first item is an addr for the created actor,
+    /// and the second item is a stream intended to be set as part of the
+    /// response via `HttpResponseBuilder::streaming()`.
+    pub fn create_with_codec_addr<S>(
```

Comment:
is adding this new standalone fn necessary anymore?

Reply:
I've moved it inside `WsResponseBuilder` since it's used only there internally. Is this what was expected?